Dockerfile

```diff
@@ -89,7 +93,5 @@ ENV SNUBA_RELEASE=$SNUBA_VERSION_SHA \
     UWSGI_DOGSTATSD_EXTRA_TAGS=service:snuba
 
 EXPOSE 1218
-
-COPY docker_entrypoint.sh ./
```

Comment:
Is this related to the change in the PR ?

Reply:
It's not necessary, since everything is copied earlier.